go.mod

```diff
@@ -129,10 +129,12 @@ require (
 	golang.org/x/sys v0.18.0 // indirect
 	golang.org/x/tools v0.15.0 // indirect
 	golang.org/x/xerrors v0.0.0-20231012003039-104605ab7028 // indirect
+	google.golang.org/genproto/googleapis/api v0.0.0-20240412170617-26222e5d3d56 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
 replace (
+	github.com/bucketeer-io/bucketeer/proto v0.0.0 => ./proto // TODO: remove this line when the proto module is merged
```

Comment:
I think we don't need to remove this line.

When we're working on local development, we update the `proto` module directly. If we remove this line, we'll need to publish the `proto` module.